core/core.go

```diff
@@ -565,3 +565,22 @@ func getData[T any](api *DefaultAPI, token string, url string) (T, error) {
 
 	return data, nil
 }
+
+const maxBytesLimit int64 = 1024 * 1024 * 10 // 10MB
+
+// re-implementation of io.ReadAll with max limit
```

Comment:
I think doc comments in go should start with the method name

Reply:
that's right, fixed